dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferBasics.cs

```diff
@@ -50,9 +68,11 @@
 
 PressEnter();
 
-
 // Download a single file from an S3 bucket.
-var keyName = "FileToDownload.docx";
+DisplayTitle("Download a single file");
+Console.WriteLine("Now we will download a single file from an Amazon S3 bucket.");
```

Comment:
Here and throughout, it's OK to use "S3 bucket" - the resource doesn't require the term Amazon.
"Now we'll download a single file from an S3 bucket."

Reply:
Fixed.